keccak256/src/permutation/rho_checks.rs

```diff
@@ -255,15 +243,20 @@ impl<F: Field> LaneRotateConversionConfig<F> {
         ),
         Error,
     > {
+        let rotation = {
+            let x = lane_idx / 5;
+            let y = lane_idx % 5;
+            ROTATION_CONSTANTS[x][y]
+        };
         let (conversions, special) = RhoLane::new(
-            f_to_biguint(lane_base_13.value().copied().unwrap_or_default()),
-            self.rotation,
+            f_to_biguint(*lane_base_13.value().unwrap_or(&F::zero())),
```

Comment:
Is the old `lane_base_13.value().copied().unwrap_or_default()` not working?

Reply:
Ohh it is. Not sure why I changed it. Let me undo!